DXRMapFixups/DeusEx/Classes/DXRFixupVandenberg.uc

```diff
@@ -98,8 +101,20 @@ function PreFirstEntryMapFixes()
             foreach AllActors(class'#var(prefix)BlackHelicopter',jock,'Helicopter'){break;}
             hoverHint = class'DXRTeleporterHoverHint'.static.Create(self, "", jock.Location, jock.CollisionRadius+5, jock.CollisionHeight+5, exit.Name);
             hoverHint.SetBaseActor(jock);
+        }
 
+        foreach AllActors(class'#var(prefix)SecurityBot2',sb,'enemy_bot') {
```

Comment:
```suggestion
        foreach AllActors(class'#var(prefix)Robot', rb,'enemy_bot') {
```

Then you won't need 2 loops, just a single loop. This is similar the way the vanilla Mission12.uc looks for these bots. You'll also only need 1 variable for the robot instead of 1 per type.

Reply:
The bots have separate hints based on their type. The alternative would be to check the class inside the `foreach` but I don't know if that would be any better.

Reply:
Actually, that would require an extra int to keep track of the different bot numbers.

Reply:
Ah I see. I think it's slightly better as a single loop yea. Unreal Engine uses a linked list of actors, so even though you have the native code handling the filtering by class, it still has to loop through the entire list and check everything, so doing it twice isn't ideal.

Reply:
As a micro-optimization enthusiast, I'm convinced.